dbcrossbarlib/src/drivers/postgres_shared/create_table_sql.rustpeg

```diff
@@ -15,7 +15,7 @@ use crate::schema::Srid;
 
 /// A `CREATE TABLE` expression.
 pub create_table -> PgCreateTable
-    = ws? "CREATE"i ws "TABLE"i ws name:identifier ws? "("
+    = ws? "CREATE"i (ws "UNLOGGED"i)? ws "TABLE"i ws name:identifier ws? "("
```

Comment:
```suggestion
    = ws? "CREATE"i ws ("UNLOGGED"i ws)? "TABLE"i ws name:identifier ws? "("
```

Not that this matters, but I think that if we write it like this, we can avoid a backtrack during parsing. This doesn't make a difference here, and PEG is pretty robust in the face of simple backtracking, but it's a good habit.